sippy-ng/src/bugs/BugButton.js

```diff
@@ -14,26 +14,28 @@ export default function BugButton(props) {
   const classes = useStyles()
   const [open, setOpen] = useState(false)
 
-  const text = `The following test is failing:
+  const text = `
+The following test is failing more than expected:
   
-  ${props.testName}
+{code:none}${props.testName}{code}
   
-Additional context here:
-  
-  ${document.location.href}`
+See the [sippy test details|${document.location.href}] for additional context.
```

Comment:
Awesome! Only nit I have is do we want to reference `sippy` here or `Component Readiness`?  One and the same at the end of the day but we tend to refer to them differently.